gremlin-go/build/generate.groovy

```diff
@@ -0,0 +1,167 @@
+import org.apache.tinkerpop.gremlin.groovy.jsr223.GremlinGroovyScriptEngine
+import org.apache.tinkerpop.gremlin.groovy.jsr223.ast.VarAsBindingASTTransformation
+import org.apache.tinkerpop.gremlin.groovy.jsr223.ast.RepeatASTTransformationCustomizer
+import org.apache.tinkerpop.gremlin.groovy.jsr223.GroovyCustomizer
+import org.codehaus.groovy.control.customizers.CompilationCustomizer
+import org.apache.tinkerpop.gremlin.features.FeatureReader
+
+import javax.script.SimpleBindings
+
+import static org.apache.tinkerpop.gremlin.process.traversal.AnonymousTraversalSource.traversal
+
+// file is overwritten on each generation
+radishGremlinFile = new File("${projectBaseDir}/gremlin-go/cucumber/gremlin.go")
+
+// assumes globally unique scenario names for keys with list of Gremlin traversals as they appear
+gremlins = FeatureReader.parse("${projectBaseDir}")
+
+gremlinGroovyScriptEngine = new GremlinGroovyScriptEngine(new GroovyCustomizer() {
+    public CompilationCustomizer create() {
+        return new RepeatASTTransformationCustomizer(new VarAsBindingASTTransformation())
+    }
+})
+translator = GolangTranslator.of('g')
+g = traversal().withEmbedded(EmptyGraph.instance())
+bindings = new SimpleBindings()
+bindings.put('g', g)
+
+radishGremlinFile.withWriter('UTF-8') { Writer writer ->
+    writer.writeLine('/*\n' +
+            'Licensed to the Apache Software Foundation (ASF) under one\n' +
+            'or more contributor license agreements.  See the NOTICE file\n' +
+            'distributed with this work for additional information\n' +
+            'regarding copyright ownership.  The ASF licenses this file\n' +
+            'to you under the Apache License, Version 2.0 (the\n' +
+            '"License"); you may not use this file except in compliance\n' +
+            'with the License.  You may obtain a copy of the License at\n' +
+            '\n' +
+            'http://www.apache.org/licenses/LICENSE-2.0\n' +
+            '\n' +
+            'Unless required by applicable law or agreed to in writing,\n' +
+            'software distributed under the License is distributed on an\n' +
+            '"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n' +
+            'KIND, either express or implied.  See the License for the\n' +
+            'specific language governing permissions and limitations\n' +
+            'under the License.\n' +
+            '*/\n')
+
+    writer.writeLine("\n//********************************************************************************")
+    writer.writeLine("//* Do NOT edit this file directly - generated by build/generate.groovy")
+    writer.writeLine("//********************************************************************************\n")
+
+    writer.writeLine(
+        'package gremlingo\n' +
+        '\n' +
+        'import (\n' +
+        '\t \"errors\"\n' +
+        '\t \"github.com/lyndonb-bq/tinkerpop/gremlin-go/driver\"\n' +
```

Comment:
i note that there are number of places that reference the "lyndonb-bq" repo. i assume those should all be changed now?

Reply:
Ah yes, that was an oversight. I will fix those.